Documentation/building/netbsd-instructions.md

```diff
@@ -0,0 +1,115 @@
+Build CoreCLR on NetBSD
+======================
+
+This guide will walk you through building CoreCLR on NetBSD.  We'll start by showing how to set up your environment from scratch.
+
+Environment
+===========
+
+These instructions are written on NetBSD 7.x on the amd64 platform, since that's the release the team uses.
+
+Older releases aren't supposed to be supported, as there is need to use modern LLVM stack (Clang, libunwind and LLDB), that is developed against the NetBSD-7.x branch. Pull Requests are welcome to address other ports (like i386 or evbarm) as long as they don't break the ability to use NetBSD/amd64.
```

Comment:
I suggest the following for the first sentence:

> Older releases aren't supported because building CoreCLR requires the modern LLVM stack (Clang, libunwind, and LLDB) that is developed against the NetBSD-7.x branch.